R/predict.R

```diff
@@ -173,7 +173,7 @@ predict.model_fit <- function(object, new_data, type = NULL, opts = list(), ...)
     linear_pred = predict_linear_pred(object = object, new_data = new_data, ...),
     hazard      = predict_hazard(object = object, new_data = new_data, ...),
     raw         = predict_raw(object = object, new_data = new_data, opts = opts, ...),
-    rlang::abort(glue::glue("I don't know about type = '{type}'"))
+    cli::cli_abort("I don't know about type = {.arg {type}}")
```

Comment:
The current phrasing of this message makes formatting a little tricky. How about:

```suggestion
    cli::cli_abort("Unknown prediction {.arg type} '{type}'.")
```